api/admin/routes.py

```diff
@@ -340,6 +348,18 @@ def stats():
     return statistics_response.api_dict()
 
 
+@app.route("/admin/quicksight_embed/<dashboard_id>")
```

Comment:
I believe we'll want to use the logical name here instead of the dashboard id.   That way the Admin UI can use hard-coded logical names which can be completely decoupled from the dashboard ARN.  (Per my comments above on the README).